### ISSUE TITLE
Game crashing when making an animate block glyph with Ars

### ISSUE THREAD

**Issue by spicyusername** [AllTheMods/ATM-8#1049]
Me and my friend are playing atm8 using essential singleplayer servers because we dont have money and whenever i try to make a glyph of animate block his game crashes. Crash report attached
[crash-2023-03-09_17.49.04-server.txt](https://github.com/AllTheMods/ATM-8/files/10938044/crash-2023-03-09_17.49.04-server.txt)

**Comment by TheDarkFlame** [comment]
relevant parts of the stack trace:
```
java.lang.NullPointerException: Ticking block entity
	at java.util.Objects.requireNonNull(Objects.java:208) ~[?:?] {re:mixin}
	at com.smashingmods.alchemylib.api.blockentity.processing.AbstractInventoryBlockEntity.getCapability(AbstractInventoryBlockEntity.java:59) ~[alchemylib-1.19.2-1.0.20.jar%23509!/:1.19.2-1.0.20] {re:classloading}
	at com.hollingsworth.arsnouveau.common.block.tile.ScribesTile.checkInventories(ScribesTile.java:127) ~[ars_nouveau-1.19.2-3.12.3.jar%23527!/:3.12.3] {re:classloading}
```

#### Individual lines
[at com.hollingsworth.arsnouveau.common.block.tile.ScribesTile.checkInventories(ScribesTile.java:127) ~[ars_nouveau-1.19.2-3.12.3.jar%23527!/:3.12.3] {re:classloading}](https://github.com/baileyholl/Ars-Nouveau/blob/1.19.x/src/main/java/com/hollingsworth/arsnouveau/common/block/tile/ScribesTile.java#L127)
```
	if (level.getBlockEntity(bPos) != null && level.getBlockEntity(bPos).getCapability(ITEM_HANDLER, null).isPresent()) {
```
[at com.smashingmods.alchemylib.api.blockentity.processing.AbstractInventoryBlockEntity.getCapability(AbstractInventoryBlockEntity.java:59) ~[alchemylib-1.19.2-1.0.20.jar%23509!/:1.19.2-1.0.20] {re:classloading}](https://github.com/SmashingMods/AlchemyLib/blob/1.19.x/src/main/java/com/smashingmods/alchemylib/api/blockentity/processing/AbstractInventoryBlockEntity.java#L57-L59)
```
    public <T> LazyOptional<T> getCapability(Capability<T> pCapability, @Nullable Direction pDirection) {
        if (pCapability == ForgeCapabilities.ITEM_HANDLER) {
            return switch (Objects.requireNonNull(pDirection)) {
```

which in turn goes beyond my current knowledge into Minecraft capability handling (forge code [here](https://docs.minecraftforge.net/en/latest/datastorage/capabilities/) Minecraft link [here](https://nekoyue.github.io/ForgeJavaDocs-NG/javadoc/1.18.2/net/minecraftforge/common/capabilities/CapabilityProvider.html#getCapability(net.minecraftforge.common.capabilities.Capability,net.minecraft.core.Direction))



TL;DR:

create an issue with the ars nouveau mod dev: https://github.com/baileyholl/Ars-Nouveau/issues/new

**Comment by Mitchell5200** [comment]
Same crash log as https://github.com/baileyholl/Ars-Nouveau/issues/893
